src/picongpu/include/fields/laserProfiles/laserWavepacket.hpp

```diff
@@ -72,23 +72,22 @@ HINLINE float3_X laserLongitudinal(uint32_t currentStep, float_X& phase)
         envelope *= math::exp(-0.5 * exponent * exponent);
     }
 
+    phase = float_X(w * runTime + LASER_PHASE);
```

Comment:
Yes, I did this on purpose to avoid any conflict wheter one should set the pase in `laserConfig.param` or in the laser implementation.

Why do we use this reference approach anyways?